src/main/kotlin/no/fdk/fdk_data_transformation_service/rdf/sparqlQueries.kt

```diff
@@ -0,0 +1,36 @@
+package no.fdk.fdk_data_transformation_service.rdf
+
+val napDatasetsQuery = """PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX dcat: <http://www.w3.org/ns/dcat#>
+
+SELECT DISTINCT ?dataset
+WHERE {
+    ?dataset a dcat:Dataset .
+    ?dataset dct:accessRights ?rights .
+    FILTER (STR(?rights) = 'http://publications.europa.eu/resource/authority/access-right/PUBLIC')
+    ?dataset dcat:theme ?theme .
+    FILTER (STR(?theme) IN ('${napThemes.joinToString("','")}'))
+}"""
```

Comment:
Litt vanskelig å vise, men noe sånt (se under). Og nei, trenger ikke encode noe.

```suggestion
val napDatasetsQuery = """
    PREFIX dct: <http://purl.org/dc/terms/>
    PREFIX dcat: <http://www.w3.org/ns/dcat#>

    SELECT DISTINCT ?dataset
    WHERE {
        ?dataset a dcat:Dataset .
        ?dataset dct:accessRights ?rights .
        FILTER (STR(?rights) = 'http://publications.europa.eu/resource/authority/access-right/PUBLIC')
        ?dataset dcat:theme ?theme .
        FILTER (STR(?theme) IN ('${napThemes.joinToString("','")}'))
    }
"""
```

Reply:
Du vil legge til en ekstra tab på hver linje? Klarer ikke helt se hvorfor det skal forhindre noe uforventet...

Reply:
Det jeg mente med "deles opp uforventet" er ren formatering - hvordan query ser ut :)

Reply:
så:
```
PREFIX dct: <http://purl.org/dc/terms/>
PREFIX dcat: <http://www.w3.org/ns/dcat#>
SELECT DISTINCT ?dataset
WHERE {
    ?dataset a dcat:Dataset .
    ?dataset dct:accessRights ?rights .
    FILTER (STR(?rights) = 'http://publications.europa.eu/resource/authority/access-right/PUBLIC')
    ?dataset dcat:theme ?theme .
    FILTER (STR(?theme) IN ('uri0','uri1','osv'))
}
```
vs
```

    PREFIX dct: <http://purl.org/dc/terms/>
    PREFIX dcat: <http://www.w3.org/ns/dcat#>
    SELECT DISTINCT ?dataset
    WHERE {
        ?dataset a dcat:Dataset .
        ?dataset dct:accessRights ?rights .
        FILTER (STR(?rights) = 'http://publications.europa.eu/resource/authority/access-right/PUBLIC')
        ?dataset dcat:theme ?theme .
        FILTER (STR(?theme) IN ('uri0','uri1','osv'))
    }
```

Reply:
Ser ikke noen grunn til å ikke gå for det første alternativet

Reply:
første er bra